### ISSUE TITLE
Turkish Translation

### ISSUE THREAD

**Issue by oz9un** [OWASP/API-Security#58]
Hey!

First of all thanks for this outstanding repository about API security.
I would be happy to contribute to the translation of this repository into **Turkish**.

![htp](https://i.giphy.com/media/FnGJfc18tDDHy/giphy.webp)

**Comment by PauloASilva** [comment]
Hi @oz9un,
Please do your magic!

I'm assigning this issue to you so that you can lead the effort in case someone wants to join you.

Cheers,
Paulo A. Silva

**Comment by MuratZamir** [comment]
Hello,

I really appreciate everything you guys have done so far for this community. I would like to contribute to the efforts of translating to Turkish, if possible.  

Thanks.

**Comment by PauloASilva** [comment]
Hi @MuratZamir,
Thanks for your kind words.

This issue is kinda of stalled: it's one year old and no content was contributed.
Would you like to take ownership of it? Please, note that this is a request to translate de 2019 edition: meanwhile the 2023 edition has been released.

Cheers,
Paulo A. Silva

**Comment by oz9un** [comment]
Yeah, you are so right @PauloASilva. We talked with @MuratZamir and decided to move on together to fasten the process. 

If there is no longer need to translate the 2019 version, we can directly work on 2023. Or we can share the work on those two. How would you like us to proceed?

Best regards,
Ozgun.

**Comment by PauloASilva** [comment]
Hi @MuratZamir,

> I believe we can start translating the 2023 edition. If any reference is made to 2019 version, we can simply translate that as well. 

Sounds like a plan!

> As a side note, I think @oz9un and I will be work on this gradually. I don't think there is an urge to finish the translation in a short amount of time. If we don't have any time restrictions, are you okay with us working on this in our own pace?

Sure thing: we are all volunteering our spare time.

Cheers,
Paulo A. Silva

**Comment by PauloASilva** [comment]
Hey folks,
How is this translation going?

Cheers,
Paulo A. Silva

**Comment by alitalhacoban** [comment]
Hi @PauloASilva,

I'd like to pick up this issue and contribute a Turkish translation of the 2023 edition, since it seems to have stalled for a while. If no one is actively working on it, I'm happy to start gradually, file by file and open a draft PR early so progress can be reviewed as I go.

Let me know if this works or if someone's already mid-way through! @oz9un, @MuratZamir

**Comment by PauloASilva** [comment]
Hi @alitalhacoban,
feel free to do it.

Check whether @oz9un and/or @MuratZamir already have some work done.

Cheers,
Paulo A. Silva

**Comment by jankesec** [comment]
Hi, since the 2023 edition's Turkish translation is now complete, I'd like to take on the 2019 edition.

@oz9un — are you still working on this, or would you mind if I pick it up?

I'll follow the same incremental approach used for the 2023 effort (#153): translate section by section and open a draft PR early so progress can be reviewed as I go.

**Comment by ErezYalon** [comment]
Resolved by #167 — Turkish (2019) translation merged into develop. Thank you @jankesec for the thorough, complete translation!

Since the 2023 edition is now the current one, if you're looking for more to do we'd love your continued help there too — your #166 (translating the GUID/authorization note into Turkish for 2023) is still open and would be great to see through.
